packages/ibm-products/src/components/PageHeader/PageHeader.stories.jsx

```diff
@@ -644,32 +644,34 @@ const Template = ({
   return (
     <>
       <style>{`.${carbonPrefix}--modal { opacity: 0; }`};</style>
-      <ContainerDivOrTabs
-        className={`${storyClass}__content-container`}
-        tabIndex={0}
-        navigation={navigation}
-      >
-        <PageHeader
-          {...props}
-          {...getNavProps(navigation)}
-          title={
-            title?.onSave
-              ? {
-                  ...title,
-                  text: titleText,
-                  onChange: handleTitleChange,
-                  onSave: handleTitleSave,
-                  onCancel: handleTitleCancel,
-                }
-              : title
-          }
+      <div className={`${storyClass}__content-container`}>
```

Comment:
Can we just use the `ContainerDivOrTabs` component here instead of adding the extra `div` with the same class?

Reply:
hi matt, the main issue is with the `<Tabs>` which doesn't take any props to set on it, hence ill be wrapping up the tabs with a div and add the classname to it to reduce the redundancy of classname on the other condition.